templates/webapps/galaxy/admin/index.mako

```diff
@@ -46,13 +46,30 @@
     <div class="unified-panel-body">
         <div class="toolMenu">
             <div class="toolSectionList">
+                    <div class="toolSectionTitle">Repositories</div>
+                    <div class="toolSectionBody">
+                        <div class="toolSectionBg">                        
+                        %if trans.app.tool_shed_registry and trans.app.tool_shed_registry.tool_sheds:
+                            <div class="toolTitle"><a href="${h.url_for( controller='admin_toolshed', action='browse_tool_sheds' )}" target="galaxy_main">Search Tool Shed</a></div>
```

Comment:
Could "server" possibly be split into "server" and "tools"?
1. jun. 2015 10.46 skrev "Nicola Soranzo" <notifications@github.com>:

> In templates/webapps/galaxy/admin/index.mako
> <https://github.com/galaxyproject/galaxy/pull/302#discussion_r31438526>:
>
> > @@ -46,13 +46,30 @@
> >      <div class="unified-panel-body">
> >          <div class="toolMenu">
> >              <div class="toolSectionList">
> > +                    <div class="toolSectionTitle">Repositories</div>
> > +                    <div class="toolSectionBody">
> > +                        <div class="toolSectionBg">
> > +                        %if trans.app.tool_shed_registry and trans.app.tool_shed_registry.tool_sheds:
> > +                            <div class="toolTitle"><a href="${h.url_for( controller='admin_toolshed', action='browse_tool_sheds' )}" target="galaxy_main">Search Tool Shed</a></div>
>
> Should this still be "Search Tool Shed*s*"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/galaxyproject/galaxy/pull/302/files#r31438526>.
>